lib/server.go

```diff
@@ -252,33 +254,24 @@ func (srv *Server) BroadcastTransaction(txn *MsgDeSoTxn) ([]*MempoolTx, error) {
 		return nil, errors.Wrapf(err, "BroadcastTransaction: ")
 	}
 
-	// At this point, we know the transaction has been run through the mempool.
-	// Now wait for an update of the ReadOnlyUtxoView so we don't break anything.
-	srv.mempool.BlockUntilReadOnlyViewRegenerated()
-
```

Comment:
I think we want to keep this for PoW for now.

Reply:
Correct. I had to move this call into the `srv._addNewTxn(...)` function where it's only called if we're on PoW